apps/test/unit/code-studio/progressReduxTest.js

```diff
@@ -407,63 +409,90 @@ describe('progressReduxTest', () => {
         const status = statusForLevel(level, levelProgress);
         assert.strictEqual(status, LevelStatus.locked);
       });
-    });
```

Comment:
Some of the it statements that were supposed to be under the describe for this method were not so I fixed that here. The next 4 changes are all just indenting to get the correct nesting. The real additions are at the bottom